tool/gravity/cli/register.go

```diff
@@ -133,17 +133,29 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.RemoveCmd.Force = g.RemoveCmd.Flag("force", "Force removal of offline node").Bool()
 	g.RemoveCmd.Confirm = g.RemoveCmd.Flag("confirm", "Do not ask for confirmation").Bool()
 
-	g.PlanCmd.CmdClause = g.Command("plan", "Display a plan for an ongoing operation")
-	g.PlanCmd.Init = g.PlanCmd.Flag("init", "Initialize operation plan").Bool()
-	g.PlanCmd.Sync = g.PlanCmd.Flag("sync", "Sync the operation plan from etcd to local store").Hidden().Bool()
-	g.PlanCmd.Output = common.Format(g.PlanCmd.Flag("output", "Output format for the plan, text, json or yaml").Short('o').Default(string(constants.EncodingText)))
-	g.PlanCmd.OperationID = g.PlanCmd.Flag("operation-id", "ID of the operation to display the plan for. It not specified, the last operation plan will be displayed").String()
-
-	g.RollbackCmd.CmdClause = g.Command("rollback", "Rollback actions")
-	g.RollbackCmd.Phase = g.RollbackCmd.Flag("phase", "Operation phase to rollback").Required().String()
-	g.RollbackCmd.PhaseTimeout = g.RollbackCmd.Flag("timeout", "Phase rollback timeout").Default(defaults.PhaseTimeout).Hidden().Duration()
-	g.RollbackCmd.Force = g.RollbackCmd.Flag("force", "Force phase rollback").Bool()
-	g.RollbackCmd.SkipVersionCheck = g.RollbackCmd.Flag("skip-version-check", "Bypass version compatibility check").Hidden().Bool()
+	g.PlanCmd.CmdClause = g.Command("plan", "Manage ongoing operation")
+	g.PlanCmd.OperationID = g.PlanCmd.Flag("operation-id", "ID of the active operation. It not specified, the last operation will be used").String()
```

Comment:
UX Thought. How does someone find the operation-id to pass if they don't want the active operation?

Reply:
True, it's currently more of a low-level configuration. `gravity status` lists active/last operations, could be extended to output all operation with an additional flag I suppose.